frontend/src/services/Service.js

```diff
@@ -0,0 +1,77 @@
+import Fetcher from "../helpers/Fetcher";
+
+/**
+ * Default services (aka C.R.U.D.)
+ */
+class Service
+{
+    /**
```

Comment:
A rota dos params pode ser protected na classe.
route: String;

e no construtor:
constructor(route = "/") {
       this.route = route;
}

Reply:
fixed :+1: